### PR TITLE
Added warning dialog when YK with CCID disabled is inserted

### LINKED ISSUE #51: Detect NEO's with CCID disabled
CCID is disabled by default on NEO's. This can cause confusion for users of Yubico Authenticator.

Yubico Authenticator should detect that a NEO without Smart Card communication is inserted (and give the user an option to enable it). Since some users might want to just use the OTP slots (not enable CCID), there should be a _"Don't show this again"_ checkbox in the dialog window.

See [this forum topic](http://forum.yubico.com/viewtopic.php?f=26&t=2021) for discussion.

### PR DESCRIPTION
**Pull request by minisu**
Closes #51

### CODE REVIEW

File: yubioath/gui/utils.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def is_minimized(window):
```
Comment (dainnilsson) [inline yubioath/gui/utils.py]: This should probably go into yubicommons.

File: yubioath/gui/view/dialogs.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from PySide.QtGui import QCheckBox, QDialog, QGridLayout, QLabel, \
+    QDialogButtonBox
+
+ENABLE_CCID_URL = 'http://yubi.co/modeswitch'
+LABEL_TEXT = '''<b>CCID (smart card capabilities) is disabled on the inserted YubiKey.</b><br><br>
```
Comment (dainnilsson) [inline yubioath/gui/view/dialogs.py]: This string should go in messages.py

File: yubioath/gui/view/dialogs.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                Without CCID enabled, you will only be able to store 2 credentials.<br><br>
+                <a href="%s">Learn how to enable CCID</a><br>'''
+
+
+class CcidDisabledDialog(QDialog):
```
Comment (dainnilsson) [inline yubioath/gui/view/dialogs.py]: This should inherit from the Dialog class in yubicommon.

File: yubioath/gui/view/dialogs.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        label = QLabel(LABEL_TEXT % ENABLE_CCID_URL, openExternalLinks=True)
+        layout = QGridLayout()
+        layout.addWidget(label, 1, 1)
+        layout.addWidget(self.do_not_ask_again, 2, 1)
+
+        buttonBox = QDialogButtonBox(QDialogButtonBox.Ok)
+        buttonBox.accepted.connect(self.close)
+
+        layout.addWidget(buttonBox, 3, 1)
+        self.setLayout(layout)
```
Comment (dainnilsson) [inline yubioath/gui/view/dialogs.py]: Consider replacing with:

layout = QGridLayout(self)

... on line 18 instead. There are probably other occurrences of this that need addressing, I don't think I knew about that trick when I wrote most of the neoman code.

File: yubioath/gui/view/dialogs.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
+from PySide.QtGui import QCheckBox, QDialog, QGridLayout, QLabel, \
```
Comment (dainnilsson) [inline yubioath/gui/view/dialogs.py]: Lost of other files in /view/ are dialogs, it's weird that this file is named "dialogs.py"

File: yubioath/gui/__main__.py [hunk 6]
```diff
@@ -25,14 +25,17 @@
 # for the parts of OpenSSL used as well as that of the covered work.
 
 from PySide import QtGui, QtCore
+from PySide.QtGui import QMessageBox
 from yubioath import __version__ as version
+from yubioath.gui.view import dialogs
```
Comment (dainnilsson) [inline yubioath/gui/__main__.py]: Unused import

**Comment by minisu** [comment]
@dainnilsson

**Pull request merged**